runtime/testy/fake_dawn.py

```diff
@@ -50,9 +50,9 @@ def tcp_relay(port):
     s.bind((host, port))
     s.listen(1)
     conn, addr = s.accept()
+    conn.send(msg)
```

Comment:
The student upload message tells runtime to kill any running student code process as a safety measure so Dawn can scp over a new version of student code for us to run. We moved it out of the while loop so it only runs once as a command to kill any running student code which also mirrors actual dawn usage of this command.